test/unit/org/apache/cassandra/cql3/RandomSchemaV2Test.java

```diff
@@ -1,304 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.apache.cassandra.cql3;
-
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.LinkedHashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
-import java.util.Optional;
-import java.util.Set;
-import java.util.concurrent.TimeUnit;
-import java.util.stream.Collectors;
-
-import org.junit.BeforeClass;
-import org.junit.Test;
-
-import accord.utils.Gen;
-import accord.utils.Gens;
-import accord.utils.RandomSource;
-import org.apache.cassandra.config.DatabaseDescriptor;
-import org.apache.cassandra.cql3.ast.CreateIndexDDL;
-import org.apache.cassandra.cql3.ast.CreateIndexDDL.Indexer;
-import org.apache.cassandra.cql3.ast.Mutation;
-import org.apache.cassandra.cql3.ast.ReferenceExpression;
-import org.apache.cassandra.cql3.ast.Select;
-import org.apache.cassandra.cql3.ast.Symbol;
-import org.apache.cassandra.cql3.ast.TableReference;
-import org.apache.cassandra.cql3.ast.Txn;
-import org.apache.cassandra.db.Keyspace;
-import org.apache.cassandra.db.marshal.AbstractType;
-import org.apache.cassandra.db.marshal.CompositeType;
-import org.apache.cassandra.db.marshal.MapType;
-import org.apache.cassandra.db.marshal.UTF8Type;
-import org.apache.cassandra.index.Index;
-import org.apache.cassandra.index.SecondaryIndexManager;
-import org.apache.cassandra.schema.ColumnMetadata;
-import org.apache.cassandra.schema.KeyspaceMetadata;
-import org.apache.cassandra.schema.Schema;
-import org.apache.cassandra.schema.TableMetadata;
-import org.apache.cassandra.service.consensus.TransactionalMode;
-import org.apache.cassandra.service.consensus.migration.TransactionalMigrationFromMode;
-import org.apache.cassandra.transport.ProtocolVersion;
-import org.apache.cassandra.utils.AbstractTypeGenerators;
-import org.apache.cassandra.utils.CassandraGenerators;
-import org.awaitility.Awaitility;
-
-import static accord.utils.Property.qt;
-import static java.lang.String.format;
-
-public class RandomSchemaV2Test extends CQLTester
```

Comment:
This test will be replaced by `SingleNodeTableWalkerTest` in the follow up patch.  CQLTester queries the same as `NODE_LOCAL`, and some CQL operations act differently there than when coordinated... this issue impacts accord's `BEGIN TRANSACTION` which is why we keep disabling CQL features... I tried porting the model here and was bitten by this... so it really is better to avoid this pattern of test in `CQLTester` unless we do coordinated... I went down a rabbit hole of allowing coordinated execution, but that then overlaps with networking... it was honestly hard to define how this class adds value when `SingleNodeTableWalkerTest` lands as it has a larger search space, validation, etc.